pkg/api/controllers/fileshare.go

```diff
@@ -689,6 +689,36 @@ func (f *FileShareSnapshotPortal) UpdateFileShareSnapshot() {
 	}
 	snapshot.Id = id
 
+	if snapshot.Name == "" {
+		errMsg := fmt.Sprintf("empty snapshot  name is not allowed. Please give valid name.")
```

Comment:
Redundant spaces between "snapshot" and "name".
Error string should not be end with ppunctuation(https://github.com/golang/go/wiki/CodeReviewComments#error-strings)